event.go

```diff
@@ -0,0 +1,70 @@
+package containerd
+
+import "time"
+
+type EventType int
+
+func (t EventType) String() string {
+	switch t {
+	case ExitEvent:
+		return "exit"
+	case PausedEvent:
+		return "paused"
+	case CreateEvent:
+		return "create"
+	case StartEvent:
+		return "start"
+	case OOMEvent:
+		return "oom"
+	case ExecAddEvent:
+		return "execAdd"
+	}
+	return "unknown"
+}
+
+const (
+	ExitEvent EventType = iota + 1
+	PausedEvent
+	CreateEvent
+	StartEvent
+	OOMEvent
+	ExecAddEvent
+)
+
+type Event struct {
+	Timestamp  time.Time
+	Type       EventType
+	Runtime    string
+	ID         string
+	Pid        uint32
+	ExitStatus uint32
+}
+
+type EventWriter interface {
```

Comment:
Just temp, we need to decide if we are going to have a global events GRPC and get that built first then its simple for me to port over.